### PR TITLE
Likes cpmponent cleanup

### PR DESCRIPTION
**Pull request by vicliu2001**
## Description

refactored likes component code

- added contants.ts for client
- removed unnecessary files and code

## Documentation Used

Please list any documentation or resources that were referenced while making these changes.

- [ ] Documentation A
- [ ] Documentation B

## Issues Faced and Resolutions

Please describe any issues you encountered during development and how you resolved them.

- Issue: Description of the issue
    - Resolution: Steps taken to resolve the issue

## How To Test?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

## Screenshots (if applicable):

Please add any screenshots that illustrate your changes.

### CODE REVIEW

**Review by eriysd — APPROVED** [review]
lgtm!

**Review by viiiiiicccky — APPROVED** [review]
looks good!

File: client/components/likes/likes.tsx [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import { useEffect, useState } from 'react';
-import { LIKES_INCREMENT_STEP, LIKES_INCREMENT_DELAY } from '../../../server/src/utils/constants';
-import { LikesDisplay } from './likesDisplay';
+import * as Client_constants from '../..//utils/constants';
```
Comment (natapokie) [inline client/components/likes/likes.tsx:2]: extra slash?

Comment (natapokie) [reply client/components/likes/likes.tsx:2]: also (one small picky thing), let's keep our casing consistent, we'll stick to camel/pascal case for any imports/variables,
so `Client_constants` -> `ClientConstants`.

the casing is a little funky, but usually imports are with pascal case, variables are camel case and constants are in all caps with underscores.

**Review by natapokie — CHANGES_REQUESTED** [review]
looks great victor! thanks for doing the refactor, just left some very small comments :)

File: client/components/likes/likes.tsx (hunk 1 above)
Comment (vicliu2001) [reply client/components/likes/likes.tsx]: Thanks Natalie! Fixed!

**Review by natapokie — APPROVED** [review]
looks great!

**Pull request merged**